src/main/java/org/jenkinsci/plugins/docker/workflow/DockerDSL.java

```diff
@@ -55,14 +55,7 @@ public MiscWhitelist() throws IOException {
             super(new StaticWhitelist(
                     // TODO should docker-commons just get a script-security dependency and mark these things @Whitelisted?
                     "new org.jenkinsci.plugins.docker.commons.credentials.DockerRegistryEndpoint java.lang.String java.lang.String",
-                    "method org.jenkinsci.plugins.docker.commons.credentials.DockerRegistryEndpoint imageName java.lang.String",
-                    // TODO delete after using https://github.com/jenkinsci/script-security-plugin/pull/15
-                    "method java.util.concurrent.Callable call",
-                    "method groovy.lang.Closure call java.lang.Object",
-                    "method java.lang.Object toString",
-                    "method java.lang.String trim",
-                    "staticMethod org.codehaus.groovy.runtime.ScriptBytecodeAdapter compareNotEqual java.lang.Object java.lang.Object",
-                    "staticMethod org.codehaus.groovy.runtime.ScriptBytecodeAdapter compareEqual java.lang.Object java.lang.Object"));
+                    "method org.jenkinsci.plugins.docker.commons.credentials.DockerRegistryEndpoint imageName java.lang.String"));
```

Comment:
All of these whitelists are in a subsequent `script-security` release, which I needed to update to in order to update `workflow.version`, which in turn I needed to do for reasons explained above to @amuniz.